kotlin/android/app/build.gradle.kts

```diff
@@ -59,6 +59,9 @@ android {
         versionName = "1.2.1"
         multiDexEnabled = true
         testInstrumentationRunner = "dev.firezone.android.core.HiltTestRunner"
+
+        val gitSha = System.getenv("GITHUB_SHA") ?: "unknown"
```

Comment:
What about builds that don't happen on CI? Where are the builds made that get submitted to the play store for review?

Reply:
It'll be `unknown`. Playstore builds always happen via the Kotlin workflow in CI.